src/journal/journald-server.h

```diff
@@ -51,6 +51,17 @@ typedef enum SplitMode {
         _SPLIT_INVALID = -1
 } SplitMode;
 
+typedef struct JournalInfo {
+        const char *name;
+        const char *path;
+
+        JournalMetrics metrics;
+
+        uint64_t cached_space_available;
+        uint64_t cached_space_limit;
+        usec_t   cached_space_timestamp;
+} JournalInfo;
```

Comment:
"Storage" is already used by an enum (defining STORAGE_AUTO, STORAGE_PERSISTENT,...).

Let's use "JournalStorage".